app/views/episodes/_form_tags.html.erb

```diff
@@ -1,15 +1,21 @@
-<div class="col-12 mb-4">
+<div class="col-12 mb-4" data-controller="category">
   <div class="card shadow border-0">
     <div class="card-header card-header-light d-flex justify-content-between">
       <h2 class="card-title h5"><%= t(".title") %></h2>
     </div>
 
     <div class="card-body">
       <div class="form-floating input-group">
-        <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: true, allow_clear: true} %>
+        <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: true, allow_clear: true, category_target: "field"} %>
         <%= form.label :categories %>
         <%= field_help_text t(".help.categories") %>
       </div>
     </div>
+
+    <div class="card-footer" data-controller="tooltip">
+      <button type="button" class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-title="<%= t(".tooltip.adfree") %>" data-bs-placement="bottom" value="adfree" data-action="click->category#addCategory"><%= t(".label.adfree") %></button>
+      <button type="button" class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-title="<%= t(".tooltip.houseadfree") %>" data-bs-placement="bottom" value="houseadfree" data-action="click->category#addCategory"><%= t(".label.houseadfree") %></button>
+      <button type="button" class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-title="<%= t(".tooltip.paidadfree") %>" data-bs-placement="bottom" value="paidadfree" data-action="click->category#addCategory"><%= t(".label.paidadfree") %></button>
```

Comment:
Oh, one more thing ... these need to check in ruby if the episode has the tags or not.  So on initial page render ... they should be blue, if `adfree` etc is already saved to the episode.